h/config.py

```diff
@@ -30,105 +27,91 @@
+    settings_manager.set('h.bouncer_url', 'BOUNCER_URL')
+
+    settings_manager.set('h.client_url', 'CLIENT_URL')
 
     # ID for the OAuth authclient that the embedded client should use when
     # making requests to OAuth endpoints. As a public client, it does not have a
     # secret.
-    EnvSetting('h.client_oauth_id', 'CLIENT_OAUTH_ID'),
+    settings_manager.set('h.client_oauth_id', 'CLIENT_OAUTH_ID')
 
     # The list of origins that the client will respond to cross-origin RPC
     # requests from.
-    EnvSetting('h.client_rpc_allowed_origins',
-               'CLIENT_RPC_ALLOWED_ORIGINS', type=aslist),
+    settings_manager.set('h.client_rpc_allowed_origins',
+                         'CLIENT_RPC_ALLOWED_ORIGINS', type=aslist)
 
-    EnvSetting('h.db_session_checks', 'DB_SESSION_CHECKS', type=asbool),
+    settings_manager.set('h.db_session_checks', 'DB_SESSION_CHECKS', type=asbool)
 
     # Environment name, provided by the deployment environment. Please do
     # *not* toggle functionality based on this value. It is intended as a
     # label only.
-    EnvSetting('h.env', 'ENV'),
+    settings_manager.set('h.env', 'ENV')
     # Where should logged-out users visiting the homepage be redirected?
-    EnvSetting('h.homepage_redirect_url', 'HOMEPAGE_REDIRECT_URL'),
-    EnvSetting('h.proxy_auth', 'PROXY_AUTH', type=asbool),
+    settings_manager.set('h.homepage_redirect_url', 'HOMEPAGE_REDIRECT_URL')
+    settings_manager.set('h.proxy_auth', 'PROXY_AUTH', type=asbool)
     # Sentry DSNs for frontend code should be of the public kind, lacking the
     # password component in the DSN URI.
-    EnvSetting('h.sentry_dsn_client', 'SENTRY_DSN_CLIENT'),
-    EnvSetting('h.sentry_dsn_frontend', 'SENTRY_DSN_FRONTEND'),
-    EnvSetting('h.websocket_url', 'WEBSOCKET_URL'),
+    settings_manager.set('h.sentry_dsn_client', 'SENTRY_DSN_CLIENT')
+    settings_manager.set('h.sentry_dsn_frontend', 'SENTRY_DSN_FRONTEND')
+    settings_manager.set('h.websocket_url', 'WEBSOCKET_URL')
 
     # Debug/development settings
-    EnvSetting('debug_query', 'DEBUG_QUERY'),
-]
-
-
-def configure(environ=None, settings=None):
-    if environ is None:
-        environ = os.environ
-    if settings is None:
-        settings = {}
-
-    for s in SETTINGS:
-        try:
-            result = s(environ)
-        except SettingError as e:
-            log.warn(e)
+    settings_manager.set('debug_query', 'DEBUG_QUERY')
 
-        if result is not None:
-            settings.update(result)
+    if 'MANDRILL_USERNAME' in environ and 'MANDRILL_APIKEY' in environ:
+        settings_manager.set('mail.username', 'MANDRILL_USERNAME')
+        settings_manager.set('mail.password', 'MANDRILL_APIKEY')
+        settings_manager.set('mail.host', 'MANDRILL_HOST', default='smtp.mandrillapp.com')
+        settings_manager.set('mail.port', 'MANDRILL_PORT', default=587)
+        settings_manager.set('mail.tls', 'MANDRILL_TLS', default=True)
+    settings = settings_manager.settings
```

Comment:
Mentioned in another comment but this `settings = settings_manager.settings` line seems to be unnecessary, can it be deleted?